test/validStrings.mjs

```diff
@@ -7,34 +7,9 @@
 
 import assert from 'assert';
 import * as ES from '../lib/ecmascript';
-import timezoneData from 'timezones.json';
 
-const timezoneNames = new Set(timezoneData.reduce((list, entry) => list.concat(entry.utc), []));
-// Remove names that are not recognized by Node (which is sourced from CLDR)
-const bad = ['Antarctica/McMurdo', 'CST6CDT', 'EST5EDT', 'MST7MDT', 'PST8PDT'];
-bad.forEach((name) => timezoneNames.delete(name));
-
-const calendarNames = [
-  'buddhist',
-  'chinese',
-  'coptic',
-  'dangi',
-  'ethioaa',
-  'ethiopic',
-  'gregory',
-  'hebrew',
-  'indian',
-  'islamic',
-  'islamic-umalqura',
-  'islamic-tbla',
-  'islamic-civil',
-  'islamic-rgsa',
-  'iso8601',
-  'japanese',
-  'persian',
-  'roc',
-  'islamicc'
-];
+const timezoneNames = Intl.supportedValuesOf('timeZone');
```

Comment:
Is it OK that this test won't run in Node 18?  Or would we use a polyfill?  Or should we conditionally use supportedValuesOf if available, and use the old hardcoded list if not?

Reply:
Looking at what this test does, I'm not sure what value we'd get from running it in multiple versions of Node unless somehow the Temporal parsers rely on something node-version-specific?

Reply:
Makes sense to me. Do we need to do anything special so that these tests won't fail when we run Demitasse tests in earlier Node versions?

Reply:
Or should we just run Demitasse tests only in the latest Node and not bother running across multiple versions, now that Demitasse tests are only a tiny fraction of things covered by Test262?

Reply:
I don't think this particular test is worth running in earlier versions, and currently validStrings is being run separately from the other Demitasse tests: https://github.com/js-temporal/temporal-polyfill/blob/445bfd3d4efabec2d23372c9cc92f8d8619c822e/.github/workflows/test.yml#L77-L88

Once this PR lands I'll look over the remaining tests in Demitasse and see whether there's any strong reason to keep running them in older Node versions.